interfaces/Portalicious/src/app/pages/project-payments/project-payments.page.html

```diff
@@ -12,5 +12,39 @@
       (click)="onCreatePayment()"
     />
   </div>
-  <p i18n>project-payments works!</p>
+
+  @if (payments()?.length === 0) {
```

Comment:
This `if` should probably be preceded with a `@if (payments.isPending())` and maybe we show a few skeletons instead of cards in that status. Otherwise there is a risk that we show "no payments found" while we are loading.